jekyll/_posts/2021-01-19-community-survey-results-2020.md

```diff
@@ -0,0 +1,221 @@
+---
+title: "Nim Community Survey 2020 Results"
+author: The Nim Team
+---
+
+Nim community survey 2020 has been open for one month, and we have received 769 responses, which we think it is large enough sample to draw the conclusions about our users and their habits.
+Before we go into details, we would like to thank all the people who took the time to respond.
+We really appreciate the feedback!
+
+The goal of this survey was primarily to determine how our community is using Nim, in order to better understand how we should be improving it.
+
+
+
+## Do you use Nim?
+
+Based on the answer to this question, the respondents were divided in two groups and they've received separate set of questions.
+
+<img src="{{site.baseurl}}/assets/news/images/survey2020/01.png" alt="Do you use Nim?" style="width:100%"/>
+
+Approximately 3/4 of the respondents use Nim (31% frequently, 41% occasionally), while the remaining 1/4 is divided between people who never used Nim (16%) and people who stopped using Nim (11%).
+
+Of those people who *don't* use Nim, the most frequent reasons are: "Nim seems immature, not ready for production" (36%), "Nim doesn't have libraries I need" (33%), "Nim seems too risky for production" (25%), and "Nim doesn't have enough learning materials" (24%).
+
+For people who have used Nim previously but stopped the most frequent reasons were (free-form question): lack of libraries (small and not mature ecosystem), incomplete documentation, bad editor support, compiler crashes, coworkers don't use it, etc.
```

Comment:
Maybe a few words on how this compares to the previous year?

Reply:
It is similar to previous year(s). It is very hard to exactly compare since it is a free-form question.